src/display/annotation_layer.js

```diff
@@ -2044,9 +2037,7 @@ class PopupElement {
       element.addEventListener("click", this.#boundToggle);
       element.addEventListener("mouseenter", this.#boundShow);
       element.addEventListener("mouseleave", this.#boundHide);
-      if (typeof PDFJSDev !== "undefined" && PDFJSDev.test("TESTING")) {
-        element.classList.add("popupTriggerArea");
-      }
+      element.classList.add("popupTriggerArea");
```

Comment:
If we do this unconditionally now, do we still need the following line?
https://github.com/mozilla/pdf.js/blob/08f9e48280dafb7d3a672e812afd760d59613430/src/display/annotation_layer.js#L2850